src/codeflare_sdk/cluster/config.py

```diff
@@ -55,3 +55,4 @@ class ClusterConfiguration:
     openshift_oauth: bool = False  # NOTE: to use the user must have permission to create a RoleBinding for system:auth-delegator
     ingress_options: dict = field(default_factory=dict)
     ingress_domain: str = None
+    write_to_file: bool = False
```

Comment:
Can we make this an arg to the cluster.up() command instead since it isn't configuration for the actual cluster?

Reply:
The appwrapper creation logic is all handled when a user runs `cluster = Cluster(ClusterConfiguration())` so the `write_to_file` bool needs to be a config param.
https://github.com/project-codeflare/codeflare-sdk/blob/main/src/codeflare_sdk/cluster/cluster.py#L74